pasportaservo/settings/base.py

```diff
@@ -219,12 +219,23 @@ def get_env_setting(setting):
 
 # Non-Django settings:
 
-SYSTEM_LOCALE = "{lang}.{encoding}".format(lang=LANGUAGE_CODE, encoding="UTF-8")
+SYSTEM_LOCALE = f'{LANGUAGE_CODE}.{"UTF-8"}'
 try:
     locale.setlocale(locale.LC_ALL, SYSTEM_LOCALE)
 except locale.Error:
-    raise locale.Error("Could not set locale {}: make sure that it is enabled on the system.".format(SYSTEM_LOCALE))
+    raise locale.Error(f"Could not set locale {SYSTEM_LOCALE}: make sure that it is enabled on the system.")
 
+def get_current_commit():
+    import subprocess
+    p = subprocess.run(
+        ('git', 'show', '-s', '--no-color', '--format=%h %D'),
+        capture_output=True, universal_newlines=True,
+    )
+    commit_hash, refnames = p.stdout.split(' ', maxsplit=1)
+    branch = refnames.strip().replace('HEAD -> ', '').replace('HEAD, ', '').split(',')[0]
+    return (commit_hash, branch)
```

Comment:
Porinforme, ĉu vi vidis tiun [SO](https://stackoverflow.com/questions/14989858/get-the-current-git-hash-in-a-python-script)?

Reply:
Jes, sed mi lernis pli de [tiu ĉi](https://stackoverflow.com/a/11168148/) respondo kaj [tiu ĉi](https://stackoverflow.com/a/42549385/) respondo. Mi volas havi kaj la haketaĵon de la enmeto kaj la nomon de la branĉo.

Reply:
la opcio %D `refnames` donas la nomojn de ĉiuj branĉoj kies kapo estas la nuna enmeto. la nura diferenco estas ke kiam vi estas en la stato de _detached head_, HEAD ne montras al ( `->` ) sed indikas ( `,` ) la branĉon de kiu ĝi estis malligita.